spec/components/result-view/history-spec.js

```diff
@@ -0,0 +1,47 @@
+/* @flow */
```

Comment:
If you remove this line, CI will go green. Otherwise we'd need to configure `flow` properly.

Reply:
I've noticed that and did a second commit on my repo. Do you know why it doesn't show up in the pull request?